### PR TITLE
Quickstart documentation

### PR DESCRIPTION
**Pull request by ndhuutai**
### Requirements

Quickstart documentation.

### Description of the Change

Added a new Quickstart page.

### Test Plan

N/A

### Alternate Designs

N/A

### Benefits

The new Quickstart page will give new developers better overview of the development and deploy flow with Neo-One.

### Possible Drawbacks

N/A

### Applicable Issues

N/A

### CODE REVIEW

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 1]
```diff
@@ -0,0 +1,285 @@
...
+You first need a wallet and the wallet's private key and sufficient amount of NEO and GAS to deploy.
+
+### Wallet set-up:
+
+1. Go to `https://neotracker.io/`
+2. Go to `“Wallet”` tab.
+3. Click on `“New Wallet”`.
+4. Enter a password (make sure you save this password), which is used to unlock the wallet later.
+5. Click on `"Download Encrypted Key"` (this will generate a `.txt` file that can be used to unlock your wallet via `“Keystore File”` option).
+6. Save the private key shown on screen locally and secure it.
```
Comment (spencercorwin) [inline packages/neo-one-website/docs/0-installation/0-quick-start.md:209]: Save your private key somewhere secure.

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 2]
```diff
@@ -0,0 +1,285 @@
...
+7. Click continue to view your wallet.
```
Comment (spencercorwin) [inline packages/neo-one-website/docs/0-installation/0-quick-start.md:210]: Click `Continue` to view your wallet.

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 3]
```diff
@@ -0,0 +1,285 @@
...
+8. (Optional) Generate a PDF by clicking on “Print paper wallet”
```
Comment (spencercorwin) [inline packages/neo-one-website/docs/0-installation/0-quick-start.md:211]: "...a PDF of your wallet..."

Comment (spencercorwin) [reply packages/neo-one-website/docs/0-installation/0-quick-start.md:211]: Surround "Print paper wallet" in backticks like you do everywhere else: `"Print paper wallet"`.

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 4]
```diff
@@ -0,0 +1,285 @@
...
+9. (Optional) Scroll down to `“Details”` when viewing your wallet to access the options of the previous steps (such as to view your private key or download keystore file).
```
Comment (spencercorwin) [inline packages/neo-one-website/docs/0-installation/0-quick-start.md:212]: "...or download a keystore file..."

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 5]
```diff
@@ -0,0 +1,285 @@
...
+
+### Migration File:
+
+Create a `migration.ts` (`migration.js` if using JS) under `neo-one` folder.
```
Comment (spencercorwin) [inline packages/neo-one-website/docs/0-installation/0-quick-start.md:216]: "Create a `migration.ts` (`migration.js` if using JS) at the path specified in your NEO•ONE configuration file".

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 6]
```diff
@@ -0,0 +1,285 @@
...
+
+::: warning
+
+Tip
+
+VSCode might display single child folders in "compact form" and prevent you from creating a new file under neo-one. Disable "compact folder" in "settings" if needed.
```
Comment (spencercorwin) [inline packages/neo-one-website/docs/0-installation/0-quick-start.md:222]: Change "VSCode" to "Visual Studio Code" to be as clear as possible.

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 7]
```diff
@@ -0,0 +1,285 @@
...
+Limited to 1000 NEO and 1000 GAS per day.
+
+If you need more than that. You must apply through Neo website. Please follow the instructions here: https://docs.neo.org/docs/en-us/network/testnet.html#applying-for-test-coin-from-neo-website
+
+::: warning
+
+Note
+
+- It costs about 500 GAS to deploy a simple contract. Checkout [System Fees](https://docs.neo.org/docs/en-us/sc/fees.html) for more details.
+- Test coins only show on TestNet. Use https://testnet.neotracker.io to explore transactions and use the wallet on TestNet.
```
Comment (spencercorwin) [inline packages/neo-one-website/docs/0-installation/0-quick-start.md:257]: "You should then be able to see the assets in your wallet at https://testnet.neotracker.io"

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 8]
```diff
@@ -0,0 +1,285 @@
...
+### Deploy to TestNet:
+
+```bash
+yarn neo-one deploy
+```
+
+### Deploy to MainNet (or others):
+
+```bash
+yarn neo-one deploy --network [network-name]
```
Comment (spencercorwin) [inline packages/neo-one-website/docs/0-installation/0-quick-start.md:270]: "yarn neo-one deploy --network <network>"

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 9]
```diff
@@ -0,0 +1,285 @@
...
+```
+
+Where `network` is one of the keys provided by your configuration (`.neo-one.config.ts`) under `networks` property. By default yarn neo-one deploy will use the test key.
```
Comment (spencercorwin) [inline packages/neo-one-website/docs/0-installation/0-quick-start.md:273]: "... under the `networks` property."

"By default `neo-one deploy` will use the `test` key."

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 10]
```diff
@@ -0,0 +1,285 @@
...
+
+::: warning
+
+Note
+
+We **HIGHLY** recommend deploying to both a local private network and the Neo TestNet before attempting to deploy to the MainNet.
+
+:::
+
+## Explore with Neo Tracker
```
Comment (spencercorwin) [inline packages/neo-one-website/docs/0-installation/0-quick-start.md:283]: "...with NEO Tracker"

NEO Tracker, our web app, is capitalized

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 11]
```diff
@@ -0,0 +1,285 @@
...
+
+If you deployed in TestNet, you can check out https://testnet.neotracker.io/. Otherwise, use https://neotracker.io.
```
Comment (spencercorwin) [inline packages/neo-one-website/docs/0-installation/0-quick-start.md:285]: "If your deployment was successful you should be able to find your contract at https://testnet.neotracker.io/browse/contract/1"

**Review by spencercorwin — CHANGES_REQUESTED** [review]

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 12]
```diff
@@ -0,0 +1,285 @@
...
+
+:::
+
+::: warning
+
+Tip
+
+- If you want to learn the concepts from the ground up and learn by doing, check out our [Tutorial](/tutorial) and reference the concepts listed in [Docs](/docs/hello-world) when something is not clear.
+
+- If you want to learn with a real-world example, check out our [Course](/course) which you can follow and see how things are tested. Also keep [Docs](/docs/hello-world) page handy for reference.
```
[the remark this answers is not in the document]
Comment (ndhuutai) [reply packages/neo-one-website/docs/0-installation/0-quick-start.md:20]: Changed. Thanks!

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 13]
```diff
@@ -0,0 +1,285 @@
...
+
+---
+
+[[toc]]
+
+---
+
+## Installations
+
+1. Install [Node](https://nodejs.org) >= 10.16.0 (Latest version recommended)
```
[the remark this answers is not in the document]
Comment (ndhuutai) [reply packages/neo-one-website/docs/0-installation/0-quick-start.md:34]: Modified. Thx.

File: packages/neo-one-website/docs/0-installation/0-quick-start.md [hunk 14]
```diff
@@ -0,0 +1,285 @@
...
+yarn add @neo-one/cli @neo-one/client @neo-one/smart-contract @neo-one/smart-contract-test @neo-one/smart-contract-lib @neo-one/smart-contract-typescript-plugin
+```
+
+```bash
+npm install @neo-one/cli @neo-one/client @neo-one/smart-contract @neo-one/smart-contract-test @neo-one/smart-contract-lib @neo-one/smart-contract-typescript-plugin
+```
+
+4. Run `yarn neo-one init` or `npx neo-one init`
+
+The command above generates a sample `HelloWorld.ts` smart contract, a sample test for the contract `HelloWorld.test.ts`, a config file `.neo-one.config.ts`, and `neo-one` folder with important modules.
```
[the remark this answers is not in the document]
Comment (ndhuutai) [reply packages/neo-one-website/docs/0-installation/0-quick-start.md:65]: Modified. Thx.

File: packages/neo-one-website/docs/0-installation/0-quick-start.md (hunk 8 above)
Comment (spencercorwin) [reply packages/neo-one-website/docs/0-installation/0-quick-start.md]: Whoops. My comment supposed to say `yarn neo-one deploy --network <network>`

**Pull request closed without merging**